tests/testdata/test_addons/package_global_config_everything_uccignore/globalConfig.json

```diff
@@ -1369,18 +1370,18 @@
                     "options": {
                         "items": [
                             {
-                                "value": "Update",
-                                "label": "update"
+                                "value": "update",
+                                "label": "Update"
                             },
                             {
-                                "value": "Delete",
-                                "label": "delete"
+                                "value": "delete",
+                                "label": "Delete"
                             }
                         ]
                     },
                     "help": "Select the action you want to perform",
                     "required": true,
-                    "defaultValue": "two"
```

Comment:
Previously - default value was something different from options.items.value and it built fine - should we change it as well (after this PR)? We should not build anything if the default value is not part of the options.

Reply:
I'll create a ticket for us to make sure that the default value is present in the `options.items.value` so it can be actually selected in the UI by default.

In this PR - we are good.